spec/src/main/asciidoc/required-apis.asciidoc

```diff
@@ -94,69 +93,69 @@ Use of implementations beyond Jakarta Annotations 1.3 in MicroProfile is allowed
  - https://github.com/eclipse-ee4j/common-annotations-api
 
 [[mp-config]]
-=== Eclipse MicroProfile Config 2.0-M1
+=== MicroProfile Config 2.0-RC1
 
-Eclipse MicroProfile Config provides applications and microservices means to obtain configuration properties through several environment-aware sources both internal and external to the application and made available through dependency injection or lookup.
+MicroProfile Config provides applications and microservices means to obtain configuration properties through several environment-aware sources both internal and external to the application and made available through dependency injection or lookup.
 
  - http://microprofile.io/project/eclipse/microprofile-config
- - https://github.com/eclipse/microprofile-config/releases/tag/2.0-M1
+ - https://github.com/eclipse/microprofile-config/releases/tag/2.0-RC1
```

Comment:
Specific references like this to the github repo tags might have to be with the RC tag, since the real release doesn't exist yet...  Just depends on how accurate one wants to be with the documentation.

Reply:
Fixed